pkg/controllers/externalsecret/externalsecret_controller_test.go

```diff
@@ -45,7 +45,7 @@ var (
 	fakeProvider   *fake.Client
 	metric         dto.Metric
 	metricDuration dto.Metric
-	timeout        = time.Second * 10
+	timeout        = time.Second * 20
```

Comment:
[The same with the PushSecret](https://github.com/external-secrets/external-secrets/pull/2501#discussion_r1264349808), and it seems some tests take more than 10 seconds...